frontend/src/pages/student/Progress.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import DoneIcon from '@material-ui/icons/Done';
+import ClearIcon from '@material-ui/icons/Clear';
+import RestoreIcon from '@material-ui/icons/Restore';
+
+function createRow(courseName, lessonName, category) {
+  return { courseName, lessonName, category };
+}
+
+function getIcon(category) {
+  if (category === 'progress') {
+    return <RestoreIcon style={{ color: 'orange' }} />;
+  } else if (category === 'completed') {
+    return <DoneIcon style={{ color: 'green' }} />;
+  } else {
```

Comment:
In principe heb je hier de else statement niet nodig omdat hiervoor returns worden gedaan met de value en de code tot hier dan unreachable zou zijn. Misschien is het mooier om dit in lijn te trekken met andere plekken in de code en hier een switch te gebruiken (hoeft niet uiteraard)

Reply:
ah right is wel een goeie, maak ik wel een switch van